src/app/space/create/apps/models/network-stat.ts

```diff
@@ -0,0 +1,3 @@
+import { Stat } from './stat';
+
+export declare interface NetworkStat extends Stat {}
```

Comment:
I'd like the NetworkStat and it's corresponding UI to be dropped completely for the time being. Once the UX is clarified it can be added; at the moment the concept of utilization bar is bogus since there is no such thing as a maximum limit on network usage in OpenShift.